css/style.css

```diff
@@ -1,19 +1,171 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&family=Rajdhani:wght@300&display=swap');
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
```

Comment:
Fine to experiment in this context, but remember when copying an actual design, you will be expected to use the required font - in this case, Roboto.